### PR TITLE
Add Node.js deploy with Docker

### PR DESCRIPTION
**Pull request by matteocontrini**
I have read and understood the Contributor's Certificate of Origin available at the end of 
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Matteo Contrini <matteo@contrini.it>

### CODE REVIEW

File: tutorials/deploy-nodejs-with-docker/01.en.md [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+docker --version
+docker ps
+docker-compose --version
+```
+
+If you're not seeing any errors or warnings, you're good to go.
+
+## Step 5 - Running the container with Docker Compose
+
+Create a file called `docker-compose.yml` with the following content on your server:
```
Comment (n-se) [inline tutorials/deploy-nodejs-with-docker/01.en.md]: You explained above about using multiple tags. Can you also explain here how to use a specific image on a specific tag?

File: tutorials/deploy-nodejs-with-docker/01.en.md [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+
+## Step 6 - Enable automated builds (optional)
+
+Fantastic, you have now deployed the first release of your *containerized* Node.js application.
+
+If your application code is hosted in a git repository on GitHub or BitBucket, I have great news for you: the Docker build process can be automated in the cloud for free by using Docker Hub.
+
+If you go in the "Builds" tab of your image on Docker Hub, you'll be prompted with a similar page:
+
+![Docker Hub automated build pages](C:\Projects\hetzner-community-content\tutorials\deploy-nodejs-with-docker\docker-hub-automated-builds-1.png)
```
Comment (n-se) [inline tutorials/deploy-nodejs-with-docker/01.en.md]: The image "link" here does not work, you need to reference your uploaded images.

Comment (matteocontrini) [reply tutorials/deploy-nodejs-with-docker/01.en.md]: Oops, sorry, that was my IDE's fault 😁 should I just put the file name?

Comment (n-se) [reply tutorials/deploy-nodejs-with-docker/01.en.md]: Yes, please.

**Pull request merged**